src/lmql/runtime/output_writer.py

```diff
@@ -90,7 +88,7 @@ async def add_interpreter_head_state(self, variable, head, prompt, where, trace,
                     self.last_value = value
                 return
             
-            os.system("clear")
+            sys.stderr.write('\033c', flush=True) # clear screen
```

Comment:
This raises an error for me. Seems like a separate .flush() call would do the trick, instead of the kwarg?